### PR TITLE
features.rhelah.atomic: add many more images into scanner

### PR DESCRIPTION
**Pull request by chuanchang**
Signed-off-by: Alex Jia ajia@redhat.com

### CODE REVIEW

**Comment by miabbott** [comment]
I'll take a look at this after the most recent RHELAH release cycle.

File: features/rhelah/atomic/atomic_scanner.feature [hunk 1]
```diff
@@ -90,6 +90,68 @@ Background: Atomic hosts are discovered
...
+
+  @pull_rhel6_image
+  Scenario: 26. Pull latest rhel6 image from repository
+       When atomic update latest "registry.access.redhat.com/rhel6" from repository
+       Then Check whether "rhel6" is installed
+
+  @pull_rhel_tools_image
+  Scenario: 27. Pull latest rhel7/rhel-tools image from repository
+       When atomic update latest "registry.access.redhat.com/rhel7/rhel-tools" from repository
+       Then Check whether "rhel-tools" is installed
```
Comment (miabbott) [inline features/rhelah/atomic/atomic_scanner.feature]: Could all these scenarios be condensed into a single scenario with an outline?

See  http://pythonhosted.org/behave/gherkin.html#scenario-outlines

```
Scenario: 17. Pull latest <image> image from repository
       When atomic update latest "registry.access.redhat.com/<image>" from repository
       Then Check whether "<image>" is installed

       Examples: Images
       | image            |
       | rhel7/rhel-tools |
       | rhel7/rsyslog    |
       | rhel7/sadc       |
```

**Review by miabbott — COMMENTED** [review]
Sorry I took so long to have a look.  I didn't realize the changes were so simple.

File: features/rhelah/atomic/atomic_scanner.feature (hunk 1 above)
Comment (chuanchang) [reply features/rhelah/atomic/atomic_scanner.feature]: Yeah, I have a new commit to add matrix for this, thanks for your reminder :)

**Review by miabbott — APPROVED** [review]
LGTM

**Pull request merged**